tests/integration/cmd_push_test.go

```diff
@@ -118,7 +118,8 @@ var _ = Describe("odo push command tests", func() {
 			// create a file and folder then push
 			err := os.MkdirAll(filepath.Join(commonVar.Context, "tests"), 0750)
 			Expect(err).To(BeNil())
-			_, err = os.Create(filepath.Join(commonVar.Context, "README.md"))
+			testFile, err := os.Create(filepath.Join(commonVar.Context, "README.md"))
+			testFile.Close()
```

Comment:
Why do you close the file as soon as it is created?

Reply:
yea, do you mean `defer testFile.Close()`

Reply:
> Why do you close the file as soon as it is created?

we are not writing anything in the file, so just closed it.

> yea, do you mean
yeah, forgot to added `defer`

Reply:
@anandrkskd Could you please add `defer` here

Reply:
No, the `defer` is not necessary as we don't want to use the file after this point. But you should check the err before using `Close` on `testFile`